src/OSPSuite.Presentation/Presenters/Charts/DataBrowserPresenter.cs

```diff
@@ -93,12 +104,14 @@ public interface IDataBrowserPresenter : IPresenter<IDataBrowserView>, IPresente
 
    public class DataBrowserPresenter : PresenterWithColumnSettings<IDataBrowserView, IDataBrowserPresenter>, IDataBrowserPresenter
    {
+      public Cache<string, List<string>> OutputMappingNames { get; set; } = new Cache<string, List<string>>();
       private readonly Cache<DataColumn, DataColumnDTO> _dataColumnDTOCache = new Cache<DataColumn, DataColumnDTO>(x => x.DataColumn, x => null);
       private readonly List<DataColumn> _allDataColumns = new List<DataColumn>();
       private Func<DataColumn, PathElements> _displayQuantityPathDefinition;
+      private bool _linkMappedOutputs;
```

Comment:
not obbvioius that this is a boolean. ALso why do you need this state? Can't you derive it from the UI?
How is this done for Used?

Reply:
renamed to _isLinkedMappedOutputs, as discussed yesterday it makes sense to keep it here